greenfield-scripts/ab2d-greenfield-secrets.py

```diff
@@ -0,0 +1,59 @@
+# This script queries AWS for secrets starting with 'ab2d/ab2d-dev/*' and creates a new secret 'ab2d/dev/*' with same secret value
+# Note this is hardcoded for dev - need to parameterize script or hardcode values for other environments
```

Comment:
Maybe generalize this in a file like `scripts/rename-secrets.py` so that it can be reused (with modification) after the migration.